local-tests/tests/testUseCustomAuthSessionSigsToPkpSign.ts

```diff
@@ -0,0 +1,121 @@
+import {
+  LitAbility,
+  LitActionResource,
+  LitPKPResource,
+} from '@lit-protocol/auth-helpers';
+import { AuthMethodScope } from '@lit-protocol/constants';
+import { TinnyEnvironment } from 'local-tests/setup/tinny-environment';
+
+/**
+ * Test Commands:
+ * NETWORK=cayenne yarn test:local --filter=testUseCustomAuthSessionSigsToPkpSign
+ * NOT AVAILABLE IN HABANERO
+ * NETWORK=localchain yarn test:local --filter=testUseCustomAuthSessionSigsToPkpSign
+ */
+export const testUseCustomAuthSessionSigsToPkpSign = async (
+  devEnv: TinnyEnvironment
+) => {
+  const alice = await devEnv.createRandomPerson();
+
+  /**
+   * This is a custom auth method. It can be anything you want.
+   */
+  const customAuthMethod = {
+    authMethodType: 89989,
+    accessToken: 'xxx',
+  };
+
+  console.log('✅ customAuthMethod:', customAuthMethod);
+
+  const customAuthMethodOwnedPkp =
+    await alice.contractsClient.mintWithCustomAuth({
+      customAuthId: 'custom-app-user-id', // ipfs hash for the auth method
+      authMethod: customAuthMethod,
+      scopes: [AuthMethodScope.SignAnything],
+    });
+  console.log('✅ customAuthMethodOwnedPkp:', customAuthMethodOwnedPkp);
+
+  const addPermittedAuthMethodReceipt =
+    await alice.contractsClient.addPermittedAuthMethod({
+      pkpTokenId: alice.pkp.tokenId,
+      authId: 'app-id-xxx:user-id-yyy',
```

Comment:
So you're overriding the above custom auth permission? Note the Custom Auth method is already permitted since you're using `mintWithCustomAuth`